tests/integration/test_agentd/test_agentd_enrollment_params.py

```diff
@@ -110,11 +130,20 @@ def override_wazuh_conf(configuration):
     try:
         # Start Wazuh
         control_service('start', daemon='wazuh-agentd')
-    except:
+    except ValueError:
         raise Exception
```

Comment:
Remove the `try/except` block. Just run `control_service` and let run the exception if something happens.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/f3a48cc26993288a6948e8e952e9cd808228f5c5